src/GitHub.App/ViewModels/PullRequestFileViewModel.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace GitHub.ViewModels
+{
+    public class PullRequestFileViewModel : IPullRequestFileViewModel
```

Comment:
It's specifically tailored to the needs of the view, so that makes it a view model to me. In particular it wouldn't be a tree if it weren't for the fact that we can display as a tree in the view and later we might want to add things like a custom icon depending on file type etc, which makes it a VM in my mind.